re-team.json

```diff
@@ -38,7 +38,7 @@
                 ".*tfvars$"
             ],
             "matchStrings": [
-                ".*amiFilter=(?<packageName>.*?)\\n(.*currentImageName=(?<currentDigest>.*?)\\n)?(.*\\n)?.*?(?<depName>[a-zA-Z0-9-_:]*)[ ]*?[:|=][ ]*?[\\\"|\"]?(?<currentValue>ami-[a-z0-9]{17})[\\\"|\"]?.*"
```

Comment:
As a side note
original regexp contains `[\\\"|\"]`
IMO intention was to match either escaped quote or quote, but it matches backslash, pipe, or quote